test/index.spec.ts

```diff
@@ -0,0 +1,1894 @@
+'use strict';
```

Comment:
I'll merge this, but I will delete this file. I don't see a reason to duplicate the tests like this.